2022q3/wtap.adoc

```diff
@@ -0,0 +1,18 @@
+=== wtap(4) enhancement
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2022Projects/AddStaHostapAndAdhocModeToWtapWlanSimulator[Add sta, hostap and adhoc mode to wtap wlan simulator]
+
+Contact: En-Wei Wu <enweiwu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+wtap(4) is a net80211(4) Wi-Fi simulator introduced by Monthadar Al Jaberi <monthadar@gmail.com> and Adrian Chadd <adrian@FreeBSD.org> in 2012.
```

Comment:
I can't find anything related to a past, present, or future manual page for wtap(4). 

Am I missing something? (Out of scope for this report?)

Reply:
Currently no, but I'm going to create it soon.

Reply:
@rickywu0421 thank you for explaining. Mystery solved `:-)`